src/ui/marker.js

```diff
@@ -484,4 +510,61 @@ export default class Marker extends Evented {
     isDraggable() {
         return this._draggable;
     }
+
+    /**
+     * Sets the `rotation` property for pointing the marker in a particular direction
+     * @param {number} [rotation=0] Sets the direction the marker is pointing
+     * @returns {Marker} `this`
+     */
+    setRotation(rotation: number) {
+        this._rotation = rotation || 0;
+        this._update();
+        return this;
+    }
+
+    /**
+     * Returns the current rotation angle
+     * @returns {number}
+     */
+    getRotation() {
+        return this._rotation;
+    }
+
+    /**
+     * Sets the `rotateAlignment` property for pointing the marker in a particular direction
```

Comment:
reference to direction here as well. I'm not sure what the best way to say this is, but I think this isn't quite right. @andrewharvey thoughts?

Reply:
I would just use "Sets the `rotationAlignment` of the marker." Further documentation of exactly what rotationAlignment is is further up in the options and I don't think needs to be duplicated here? At least this is what other options like draggable/setDraggable do. Open to other opinions though.

Reply:
I like that approach and I totally missed that this needs to be `rotationAlignment`

Reply:
Used that wording for `setPitchAlignment` and `setRotationAlignment`, and fixed the `rotateAlignment` typo.